parquet-column/src/main/java/org/apache/parquet/schema/GroupType.java

```diff
@@ -48,6 +48,10 @@ public GroupType(Repetition repetition, String name, List<Type> fields) {
     this(repetition, name, null, fields, null);
   }
 
+  public GroupType(Repetition repetition, String name, List<Type> fields, boolean isCaseSensitive) {
```

Comment:
this should not be a property of the type itself, but a property of the selection